src/Glpi/Dashboard/Grid.php

```diff
@@ -1068,9 +1070,10 @@ public function getCardHtml(string $card_id = "", array $card_options = []): str
         Profiler::getInstance()->stop(__METHOD__ . ' get card data');
 
         if ($_SESSION['glpi_use_mode'] === Session::DEBUG_MODE) {
-            // Use the current PHP request duration as the execution time for a more accurate card loading time
-            $execution_time = Profiler::getInstance()->getCurrentDuration('php_request');
-            $html .= '<span class="debug-card">' . \htmlescape($execution_time) . 'ms</span>';
+            $html .= '<span class="debug-card">';
+            $html .= "query: " . \htmlescape(Profiler::getInstance()->getCurrentDuration('php_request')) . 'ms - ';
```

Comment:
This information does not seems accurate:

<img width="1045" height="323" alt="Image" src="https://github.com/user-attachments/assets/4720939d-a13d-4368-91f3-137ef3aeef88" />

Query times do not match what I see in the profiler.